### PR TITLE
SVGs as static files

### PR DESCRIPTION
**Pull request by arty-name**

### CODE REVIEW

File: src/components/Footer.svelte [hunk 1]
```diff
@@ -14,7 +14,9 @@
 
 <hr class="dotted-line border border-accent border-dotted bg-primary" />
 <footer class="bg-primary flex flex-col justify-center items-center pt-5 pb-6 space-y-6">
-	<p><LogoBountyManagerDesktop /></p>
+	<p>
+		<img width="307" height="62" src={LogoBountyManagerDesktop} alt="Logo" />
```
Comment (BlertaSunCacti) [inline src/components/Footer.svelte:18]: The width and height are already part of the svg file I reckon. We could therefore spare them here.

File: src/components/app-bar/AppBar.svelte [hunk 2]
```diff
@@ -93,12 +93,12 @@
...
-			><LogoBountyManagerDesktop /></button
-		>
-		<button on:click={() => goto('/curator-actions')} class="md:hidden"
-			><LogoBountyManagerMobile /></button
-		>
+		<button on:click={() => goto('/curator-actions')} class="hidden md:inline-flex">
+			<img width="307" height="62" src={LogoBountyManagerDesktop} alt="Logo" />
+		</button>
+		<button on:click={() => goto('/curator-actions')} class="md:hidden">
+			<img width="71" height="54" src={LogoBountyManagerMobile} alt="Logo" />
```
Comment (BlertaSunCacti) [inline src/components/app-bar/AppBar.svelte:100]: Same thing as above.

File: src/components/common/Dialog.svelte [hunk 3]
```diff
@@ -47,7 +47,7 @@
 			<div class="flex justify-end">
 				{#if dismissable}
 					<button class={getColorClass(textColor)} on:click={() => (open = false)}>
-						<CloseIcon />
+						<img src={CloseIcon} alt="Close icon" height="30" width="30" />
```
Comment (BlertaSunCacti) [inline src/components/common/Dialog.svelte:50]: Here too.

File: src/components/curator-actions/child-bounties/operations/AddChildBounty.svelte [hunk 4]
```diff
@@ -104,7 +104,9 @@
 				on:input={inputChange}
 			/>
 			<div class="border border-accent absolute right-9 top-9 transform -translate-y-1/2 h-6"></div>
-			<div class="absolute right-2 top-[26px]"><PolkaCoin /></div>
+			<div class="absolute right-2 top-[26px]">
+				<img src={PolkaCoin} width="20" height="20" alt="PolkaCoin" />
```
Comment (BlertaSunCacti) [inline src/components/curator-actions/child-bounties/operations/AddChildBounty.svelte:108]: Here as well.

File: src/components/curator-actions/child-bounties/operations/AssignSubCurator.svelte [hunk 5]
```diff
@@ -89,7 +89,9 @@
 				placeholder="0"
 			/>
 			<div class="border border-accent absolute right-9 top-9 transform -translate-y-1/2 h-6"></div>
-			<div class="absolute right-2 top-[26px]"><PolkaCoin /></div>
+			<div class="absolute right-2 top-[26px]">
+				<img src={PolkaCoin} width="20" height="20" alt="PolkaCoin" />
```
Comment (BlertaSunCacti) [inline src/components/curator-actions/child-bounties/operations/AssignSubCurator.svelte:93]: See above.

File: src/components/curator-actions/child-bounties/operations/BatchChildBountyCalls.svelte [hunk 6]
```diff
@@ -130,7 +130,9 @@
 				placeholder="0"
 			/>
 			<div class="border border-accent absolute right-9 top-9 transform -translate-y-1/2 h-6"></div>
-			<div class="absolute right-2 top-[26px]"><PolkaCoin /></div>
+			<div class="absolute right-2 top-[26px]">
+				<img src={PolkaCoin} width="20" height="20" alt="PolkaCoin" />
```
Comment (BlertaSunCacti) [inline src/components/curator-actions/child-bounties/operations/BatchChildBountyCalls.svelte:134]: Once more.

**Review by BlertaSunCacti — CHANGES_REQUESTED** [review]

**Comment by arty-name** [comment]
@BlertaSunCacti, it is recommended to always provide a size for an image either via such attributes or via CSS, because otherwise the browser will initially render the page with just a small placeholder, and once the image loads and its intrinsic dimensions become known, the browser will re-render everything in different sizes resulting in "jumping" content. This is called [Layout Shift](https://web.dev/articles/cls) and is an important metric of website quality.

**Review by abdulmth — APPROVED** [review]

**Review by BlertaSunCacti — APPROVED** [review]

**Pull request merged**